lib/wda/xcodebuild.js

```diff
@@ -175,6 +185,14 @@ class XcodeBuild {
       '-scheme', 'WebDriverAgentRunner',
       '-destination', `id=${this.device.udid}`,
     ];
+
+    if (this.useXCTESTRUNFile) {
+      genericArgs = [
```

Comment:
These aren't "generic arguments" anymore. I think it would make more sense to something like
```js
if (this.useXCTESTRUNFile) {
  args.push('-xctestrun', this.xctestrunFilePath);
} else {
  args.push('-project', this.agentPath, '-scheme', 'WebDriverAgentRunner');
}
args.push('-destination', `id=${this.device.udid}`);
```